nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -2591,3 +2598,7 @@ void nxagentSetCoalescence()
   nxagentChangeOption(DisplayCoalescence, timeout);
 }
 
+void nxagentShowVersionInfo()
```

Comment:
`nxagentShowVersionInfo(void)` please.

Reply:
> nxagentShowVersionInfo(void) please.

Done. Also added another commit that fixes this issues for all functions in hw/nxagent/.